jira/client.py

```diff
@@ -156,7 +156,7 @@ def _field_worker(fields=None, **fieldargs):
 
 class ResultList(list):
 
-    def __init__(self, iterable=None, _startAt=None, _maxResults=None, _total=None, _isLast=None):
+    def __init__(self, iterable=[], _startAt=0, _maxResults=0, _total=0, _isLast=None):
```

Comment:
I strongly suggest not setting `iterable` to `[]` like this but checking if it is `None` and then setting it to `[]` as needed.